runway/hooks/staticsite/auth_at_edge/templates/shared.py

```diff
@@ -0,0 +1,358 @@
+"""Shared functionality for the Auth@Edge Lambda suite."""
+
+import base64
+import json
+import logging
+import re
+import time
+import traceback
+from datetime import datetime
+from random import random
+from urllib import request  # pylint: disable=no-name-in-module
+from urllib.parse import urlencode  # pylint: disable=no-name-in-module,import-error
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_config():
+    """Retrieve the configuration variables for the Auth@Edge suite.
+
+    Lambda@Edge restricts the ability to use environment variables. This configuration
+    object is generated with hard coded values via Runway.
+    """
+    # This configuration will be replaced with dynamic values
+    # via the lambda_config.py Runway hook. Please review
+    # that file for more information.
+    config = {
+        'client_id': 'tbd',
+        'cognito_auth_domain': 'tbd',
+        'cookie_settings': {},
+        'http_headers': {},
+        'oauth_scopes': [],
+        'redirect_path_auth_refresh': 'tbd',
+        'redirect_path_sign_in': 'tbd',
+        'redirect_path_sign_out': 'tbd',
+        'user_pool_id': 'tbd',
+    }
+
+    user_pool_region = 'us-east-1'
+    region_match = re.match(
+        r"^(\\S+?)_\\S+$",
+        config['user_pool_id']
+    )
+    if region_match:
+        user_pool_region = region_match.groups()[0]
+
+    # pylint: disable=unsupported-assignment-operation
+    config['cloud_front_headers'] = as_cloud_front_headers(
+        config['http_headers']
+    )
+    # pylint: disable=unsupported-assignment-operation
+    config['token_issuer'] = 'https://cognito-idp.%s.amazonaws.com/%s' % (
+        user_pool_region,
+        config['user_pool_id']
+    )
+    # pylint: disable=unsupported-assignment-operation
```

Comment:
With the `config` ^ being a non-None object now, these "unsupported-assignment" lines should be able to be dropped